FWCore/Utilities/interface/GlobalIdentifier.h

```diff
@@ -4,6 +4,8 @@
 #include <string>
 namespace edm {
   std::string createGlobalIdentifier(bool binary = false);
-}
+
+  bool isValidGlobalIdentifier(std::string const& guid);
```

Comment:
When can we have a non-valid one? Maybe a comment here?

OK, now I see this is testing that the string matches the required format.